README.md

```diff
@@ -107,3 +107,46 @@ async function run() {
   // do stuff with the client
 }
 ```
+
+#### Creating transactions
+
+Creating transactions is a two step process. First a procedure is created, which validates the chain is likely to accept the transaction and returns a `Procedure` object. This procedure is then ran
+
+```typescript
+  /**
+   * This step performs validations, and will throw if the transaction isn't expected to proceed, e.g. `ticker` is already used
+   */
+  const createAssetProc = await polyClient.assets.createAsset({
+    name: 'My new asset'
+    ticker: 'TICKER',
+    // ... (args omitted for brevity)
+  })
+
+  /**
+   * The promise will resolve when the transaction is in a finalized block which takes on average 15 seconds. It will throw if the transaction fails to finalize
+   * For example `ticker` was claimed after the procedure was created, but before it was ran, or the signing manager didn't generate a correct signature.
```

Comment:
```suggestion
   * The promise will resolve when the transaction is in a finalized block, which takes on average 15 seconds. It will throw an error if the transaction fails to finalize.
   * For example, if the `ticker` was claimed after the procedure was created but before it was executed, or if the signing manager didn't generate a correct signature.
```